### ISSUE TITLE
Add feature to find "stale" containers

### ISSUE THREAD

**Issue by jodh-intel** [clearcontainers/runtime#630]
If the assets are updated (kernel + rootfs image), there should be a way to determine which containers that were running at the time of the asset updated are now "stale" (really need to be restarted to take advantage of the new assets for functionality/performance/security improvements).

The simplest solution is a script, but a better way is to modify `list --cc-all` to show whether the assets used for each container are current or not.

**Comment by jodh-intel** [comment]
@grahamwhaley - as discussed on #631, I know your view on making the `list` output even longer. However, the output is already wrapping for me so I'm inclined to just add the extra fields. What we can do to alleviate the problem of long output lines:

- Once #668 lands, the output of `list` will be saner as the number of columns will be fixed. It's then a simple matter of (ironically :-) running a 1-line `awk` script on the output of `list --cc-all` to pull out the fields you only want to see.

- Alternatively, we could consider a separate PR to add a new `list --format=csv` for example to make it even easier to parse.

- A potentially better alternative to `list --format=csv` might be to add a `list --separator=<char>` which would only apply to the default `table` output and would allow you to specify say a single space or a single tab to separate columns. This coupled with a `CC_RUNTIME_SEPARATOR=char` would allow you to have a persistent way not having to type lots of cli options to get the output you want.

- we could go crazy and offer a `--cc-format-string="%s\t%s ..."` type feature.

As it minimises adding extra code paths, I'm going to raise a PR that just adds the extra fields for now.

**Comment by grahamwhaley** [comment]
tailing note after merge/close...
I'm not that against the long output - just noted that it is already rather long :-)
As you've now fixed the format, we can call it 'parseable', and thus I don't think at this point in time we need to add any more format output options - let's reserve that for the future if we get any user requests for it etc.

**Comment by jodh-intel** [comment]
Agreed.
